privacyidea/static/components/config/views/config.periodictasks.list.html

```diff
@@ -20,14 +20,14 @@
                        ng-change="orderChanged(ptask)" />
             </td>
             <td>
-                <span ng-show="ptask.active"
-                      class="glyphicon glyphicon-ok"
-                      ng-click="disablePeriodicTask(ptask.id)">
-                </span>
-                <span ng-hide="ptask.active"
-                      class="glyphicon glyphicon-minus"
-                      ng-click="enablePeriodicTask(ptask.id)">
-                </span>
+                <span
```

Comment:
@NuvandaPV Honestly I think this change is not necessary. And I like the previous notation better. It worked well for quite some time.
I am not sure, if we need to define a code style guide or if we need to configure the IDE the same way, so that we beautify in the same direction!